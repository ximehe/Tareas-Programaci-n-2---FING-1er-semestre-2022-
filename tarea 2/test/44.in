# implementar reversoDeIterador
reversoDeIterador
agregarAIterador 1
reversoDeIterador
agregarAIterador 2
reversoDeIterador
reiniciarEstructuras
agregarAIterador 1
agregarAIterador 2
agregarAIterador 3
agregarAIterador 4
agregarAIterador 5
agregarAIterador 6
reversoDeIterador
reversoDeIterador
agregarAIterador 13
agregarAIterador 12
agregarAIterador 11
reversoDeIterador
Fin