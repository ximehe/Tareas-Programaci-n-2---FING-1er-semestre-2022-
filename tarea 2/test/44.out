1>#  implementar reversoDeIterador.
2>
3>Agregando 1 a it.
4>1 
5>Agregando 2 a it.
6>2 1 
7>Estructuras reiniciadas.
8>Agregando 1 a it.
9>Agregando 2 a it.
10>Agregando 3 a it.
11>Agregando 4 a it.
12>Agregando 5 a it.
13>Agregando 6 a it.
14>6 5 4 3 2 1 
15>6 5 4 3 2 1 
16>Agregando 13 a it.
17>Agregando 12 a it.
18>Agregando 11 a it.
19>11 12 13 6 5 4 3 2 1 
20>Fin.
